sdlib/national_districts.py

```diff
@@ -0,0 +1,455 @@
+                         ("TX", 19, "TX-19"),
+                         ("TX", 2, "TX-02"),
+                         ("TX", 20, "TX-20"),
+                         ("TX", 21, "TX-21"),
+                         ("TX", 22, "TX-22"),
+                         ("TX", 23, "TX-23"),
+                         ("TX", 24, "TX-24"),
+                         ("TX", 25, "TX-25"),
+                         ("TX", 26, "TX-26"),
+                         ("TX", 27, "TX-27"),
+                         ("TX", 28, "TX-28"),
+                         ("TX", 29, "TX-29"),
+                         ("TX", 3, "TX-03"),
+                         ("TX", 30, "TX-30"),
+                         ("TX", 31, "TX-31"),
+                         ("TX", 32, "TX-32"),
+                         ("TX", 33, "TX-33"),
+                         ("TX", 34, "TX-34"),
+                         ("TX", 35, "TX-35"),
+                         ("TX", 36, "TX-36"),
+                         ("TX", 4, "TX-04"),
+                         ("TX", 5, "TX-05"),
+                         ("TX", 6, "TX-06"),
+                         ("TX", 7, "TX-07"),
+                         ("TX", 8, "TX-08"),
+                         ("TX", 9, "TX-09"),
+                         ("UT", 1, "UT-01"),
+                         ("UT", 2, "UT-02"),
+                         ("UT", 3, "UT-03"),
+                         ("UT", 4, "UT-04"),
+                         ("VA", 1, "VA-01"),
+                         ("VA", 10, "VA-10"),
+                         ("VA", 11, "VA-11"),
+                         ("VA", 2, "VA-02"),
+                         ("VA", 3, "VA-03"),
+                         ("VA", 4, "VA-04"),
+                         ("VA", 5, "VA-05"),
+                         ("VA", 6, "VA-06"),
+                         ("VA", 7, "VA-07"),
+                         ("VA", 8, "VA-08"),
+                         ("VA", 9, "VA-09"),
+                         ("VT", None, "VT-00"),
+                         ("WA", 1, "WA-01"),
+                         ("WA", 10, "WA-10"),
+                         ("WA", 2, "WA-02"),
+                         ("WA", 3, "WA-03"),
+                         ("WA", 4, "WA-04"),
+                         ("WA", 5, "WA-05"),
+                         ("WA", 6, "WA-06"),
+                         ("WA", 7, "WA-07"),
+                         ("WA", 8, "WA-08"),
+                         ("WA", 9, "WA-09"),
+                         ("WI", 1, "WI-01"),
+                         ("WI", 2, "WI-02"),
+                         ("WI", 3, "WI-03"),
+                         ("WI", 4, "WI-04"),
+                         ("WI", 5, "WI-05"),
+                         ("WI", 6, "WI-06"),
+                         ("WI", 7, "WI-07"),
+                         ("WI", 8, "WI-08"),
+                         ("WV", 1, "WV-01"),
+                         ("WV", 2, "WV-02"),
+                         ("WV", 3, "WV-03"),
+                         ("WY", None, "WY-00")])
```

Comment:
this is the only really big change from bob's PR right? the merge is going to get interesting :)